source/core/owt_base/VideoFrameConstructor.cpp

```diff
@@ -62,7 +62,7 @@ bool VideoFrameConstructor::init()
     m_remoteBitrateObserver.reset(new VieRemb(Clock::GetRealTimeClock()));
     m_remoteBitrateEstimator.reset(new RemoteEstimatorProxy(Clock::GetRealTimeClock(), m_packetRouter.get()));
     m_videoReceiver.reset(new ViEReceiver(m_video_receiver.get(), m_remoteBitrateEstimator.get(), this));
-    m_videoReceiver->SetReceiveTransportSequenceNumberStatus(true, 2);
+    m_videoReceiver->SetReceiveTransportSequenceNumberStatus(true, transportccExtId);
```

Comment:
Please check the extID and only do this when extID != -1;

Same for other places that applies this ExtId.